decidim/decidim
# Confirm password validation

#### :tophat: What? Why?
This is a password confirmation alert message change to allow the alert to be more clearer according to the original issuer #11283 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #11284

#### Testing
1. Head over to Sign in/Log in
2. Click create an account
3. Fill the form
4. Purposely create different passwords in the confirmation field and hit enter
5. See the the error output has quotes

### :camera: Screenshots

:hearts: Thank you!
